src/test/regress/expected/columnar_truncate.out

```diff
@@ -268,6 +268,13 @@ set columnar.compression to default;
 CREATE USER truncate_user;
 NOTICE:  not propagating CREATE ROLE/USER commands to worker nodes
 HINT:  Connect to worker nodes directly to manually create all necessary users and roles.
+SELECT run_command_on_workers($$CREATE USER truncate_user;$$);
+      run_command_on_workers
+---------------------------------------------------------------------
+ (localhost,57637,t,"CREATE ROLE")
+ (localhost,57638,t,"CREATE ROLE")
+(2 rows)
+
```

Comment:
I was checking this as one of the few enterprise merge conflict.

In general it will be easier to create the remote users before the local users with regards to enterprise changes required for this change.

When enterprise propagates the user it is expected to be idempotent, meaning if the user already exists it configures it like it is supposed to be without error. You calling this after the remote user might be created in enterprise will return an error from the workers which changes output files here.

I will suggest to move this statement above the local creation.